lib/opte-ioctl/src/lib.rs

```diff
@@ -171,6 +172,13 @@ impl OpteHdl {
         run_cmd_ioctl(self.device.as_raw_fd(), cmd, Some(&req))
     }
 
+    /// Clear xde underlay devices.
```

Comment:
Vague architecture question: why is this code duplicated in `opte-ioctl` and `opteadm`?  It looks like `opteadm` already has a dependency on `opte-ioctl`; is there a subtle difference between `struct OpteAdm` and `struct OpteHdl`?

Reply:
Staring at it for a bit, I don't see that there *is* much reason for us to be doing this. We have a decent number of ioctls that appear in one but not the other. `OpteAdm` doesn't appear to have any methods which convert from CLI-specific datastructures, either: and if we did, there's no reason that `OpteAdm` couldn't just deref to an inner `OpteHdl`.

Opened #529.